independent-projects/resteasy-reactive/server/runtime/src/main/java/org/jboss/resteasy/reactive/server/ServerExceptionMapper.java

```diff
@@ -45,4 +46,9 @@
 public @interface ServerExceptionMapper {
 
     Class<? extends Throwable>[] value() default {};
+
+    /**
+     * The priority with which the exception mapper will be executed
+     */
+    int priority() default Priorities.USER;
```

Comment:
I must be thick, but I don't see where that is wired in by the processor?

Reply:
Good point! It had already been wired up: https://github.com/quarkusio/quarkus/blob/master/extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/main/java/io/quarkus/resteasy/reactive/server/deployment/ResteasyReactiveScanningProcessor.java#L321

Reply:
HAHAHA